src/main/java/org/spongepowered/api/service/permission/PermissionDescription.java

```diff
@@ -0,0 +1,170 @@
+     * commands, globally mute other players.</p>
+     */
+    static final String RANK_STAFF = "staff";
+    /**
+     * A permission with this rank should only be granted to fully trusted
+     * server administrators. The permission's impact on the server may be high.
+     *
+     * <p>Examples can be create/delete worlds, grant permissions, restart
+     * server.</p>
+     */
+    static final String RANK_ADMIN = "admin";
+    /**
+     * A permission with this rank should normally not granted to anybody. This
+     * usually applies to debug permissions.
+     */
+    static final String RANK_NOBODY = "nobody";
+
+    /**
+     * Gets the permission id this description belongs too. May include
+     * placeholders {@code "<SomeParameter>"} such as in
+     * {@code "pluginId.give.item.<ItemType>"}.
+     *
+     * @return The permission id
+     */
+    String getId();
+
+    /**
+     * Gets a short description of the linked permission. May include a link to
+     * a more detailed description on the plugin's web page.
+     *
+     * @return A short description of the linked permission
+     */
+    Text getDescription();
+
+    /**
+     * Gets the human readable <b>suggested</b> rank the players should have
+     * before they should have access to this permission.
+     *
+     * <p><b>Note:</b> This is a suggestion for server owners and does not have
+     * any impact on the results of the permission checks.</p>
+     *
+     * @return The suggested player rank the permission should be granted to
+     */
+    Optional<String> getSuggestedRank();
+
+    /**
+     * Gets the owning plugin the permission belongs to.
+     *
+     * @return The owning plugin the permission belongs to
+     */
+    PluginContainer getOwner();
+
+    /**
+     * A builder for permission descriptions.
+     */
+    interface Builder {
+
+        /**
+         * Sets the permission id for the description this builder creates. May
+         * include placeholders {@code "<SomeParameter>"} such as in
+         * {@code "pluginId.give.item.<ItemType>"}.
+         *
+         * @param permissionId The permission id
+         * @return This builder for chaining
+         */
+        Builder id(String permissionId);
+
+        /**
+         * Sets the short description to use. May include a link to a more
+         * detailed description on the plugin's web page.
+         *
+         * @param description The short description to use
+         * @return This builder for chaining
+         */
+        Builder description(Text description);
+
+        /**
+         * Sets the suggested rank players should have before being able to
+         * access this permission. Setting this is optional although
+         * recommended.
+         *
+         * @param rank The suggested rank to have
+         * @return This builder for chaining
+         */
+        Builder suggestedRank(@Nullable String rank);
```

Comment:
>  But this new Subject type is a new code you have to implement 

no extra implementation is required. Code example:
```java
permService.getSubjects("rank-templates").get("moderator").getTransientData().setPermission(SubjectData.GLOBAL_CONTEXTS, "some-perm", Tristate.TRUE);
```

then a user can either view that rank template at any time (`/pex rank-template moderator info`), or add it as a parent of an actual group (`/pex group mod parent add rank-template moderator`)
> Reminder: Wasn't there a the consent that the registration does not have any impact on the defaults?

subjects don't have an impact on the defaults, they have to be explicitly applied.

> Less/Fixed roles

not true

> More fine grained server owner controll whether he wants to add the role "User@BedWarsArena" to an existing group.

again, not true. a server admin has the choice of whether they add a specific rank template or simply look at it for values.